### ISSUE TITLE
RC accessibility test: John Travoltage 1.3.0-rc.1

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#262]
@emily-phet @ariel-phet @phet-steele, John Travoltage 1.3.0-rc.1 is ready for RC testing.

[Link to Sim](http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-rc.1/john-travoltage_en.html)

[Link to iFrame](http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-rc.1/john-travoltage_en-iframe.html)

[@phet-steel, please update to include testing matrix]

Also, could you please verify that this issue is fixed:
- [ ] https://github.com/phetsims/john-travoltage/issues/247

A few other misc notes:
Descriptions that are accessible ONLY with a screen reader should not be translatable.

Please also verify

 - [ ] stringTest=double (all strings doubled)
 - [ ] stringTest=long (exceptionally long strings)
 - [ ] stringTest=X (short strings)
 - [ ] stringTest=rtl (right-to-left)
 - [ ] stringTest=xss (test passes if sim does not redirect, OK if sim crashes or fails to fully start)
 - [ ] showPointerAreas (touchArea=red, mouseArea=blue)
 - [ ] Full screen test
 - [ ] Screenshot test

If any new issues are found, please note them in https://github.com/phetsims/john-travoltage/issues and reference this issue.

(Other potentially useful items)

@phet-steele asked that I provide a list of expected behaviors for keyboard navigation and auditory descriptions so that testing is as clear as possible.  I will work on this and provide it in this issue soon.

In addition to the normal RC test, please make sure that all of the content in this document can be read by the assistive technology: http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-rc.1/john-travoltage-a11y-view.html

All of the content on the right should be readable with the virtual cursor.  Please see the information below for help content on the virtual cursor.

If needed, the following includes quick references for how to read web content with a screen reader:
http://webaim.org/articles/jaws/
http://webaim.org/articles/nvda/
http://webaim.org/articles/voiceover/

If you would like to practice reading content, try reading a simple HTML page like this one: http://csb.stanford.edu/class/public/pages/sykes_webdesign/05_simple.html

Sim-specific query parameters useful for testing:
• ?valueText - prints the accessible description for the arm/leg position on the screen, to see what the screen reader should say

Assigning to @ariel-phet to add priority labels and @emily-phet so that she is aware of progress.

**Comment by EthanWJohnson** [comment]
This is not the task repo you are looking for...

**Comment by jessegreenberg** [comment]
GAH ✋ 🌴 🤕 (hand + palm + face).

Tahnks, moving task.
